build/images/scripts/install_cni

```diff
@@ -33,5 +33,8 @@ install -m 755 /opt/cni/bin/whereabouts /host/opt/cni/bin/whereabouts
 # Load the OVS kernel module
 modprobe openvswitch || (echo "Failed to load the OVS kernel module from the container, try running 'modprobe openvswitch' on your Nodes"; exit 1)
 
+# Load the WireGuard kernel module
+modprobe wireguard || (echo echo "Failed to load the WireGuard kernel module, WireGuard encryption will not be available")
```

Comment:
Could we add an argument to install_cni and perform this only when WG encryption is enabled?

Reply:
This means we need to parse the antrea-config in the init-container. I wonder whether this is necessary.

Reply:
I agree with @xliuxu, I think we can avoid it. The antrea-agent initialization will fail if the kernel module is not present / loaded anyway.

Reply:
Yeah, the problem is we do not have a separate yaml for WG.

I do not like to load WG module when not needed, but do you know it has any impact like traffic overhead, or memory consumption?

Reply:
I would imagine that there is no overhead if you don't configure any device for WireGuard and don't use Wireguard.

However, I think there have been other instances in the past where we wished we had access to the configMap in the install_cni container. I don't think there is any harm in consuming the config in the initContainer. I do feel like we should not do it from a shell script though, so if we want to go this route I suggest we write a small Go program to take care of parsing the config and taking the appropriate actions. It could be a follow-up PR though.